lib/python/pyflyby/_interactive.py

```diff
@@ -1236,7 +1236,10 @@ def new_IPdb_instance():
     pdb_class = _get_IPdb_class()
     logger.debug("new_IPdb_instance(): pdb_class=%s", pdb_class)
     color_scheme = _get_ipython_color_scheme(app)
-    pdb_instance = pdb_class(color_scheme)
+    try:
+        pdb_instance = pdb_class(completekey='tab', color_scheme=color_scheme)
+    except TypeError:
+        pdb_instance = pdb_class(completekey='tab')
```

Comment:
Given the default value for completekey is 'tab', we can skip specifying it.

Reply:
@asmeurer wrote in https://github.com/deshaw/pyflyby/issues/213#issuecomment-1372918162:

> For whatever reason `tab` was not being set as the complete key unless I passed it manually, even though it ought to be the default.